src/Processors/Formats/Impl/AvroRowOutputFormat.cpp

```diff
@@ -41,15 +41,42 @@
 #include <avro/ValidSchema.hh>
 #include <avro/Writer.hh>
 
+#include <re2/re2.h>
 
 namespace DB
 {
 namespace ErrorCodes
 {
     extern const int ILLEGAL_COLUMN;
     extern const int BAD_ARGUMENTS;
+    extern const int CANNOT_COMPILE_REGEXP;
 }
 
+class AvroSerializerTraits
```

Comment:
Put in anonymous namespace.

Reply:
@alexey-milovidov   The problem here there is a forward declaration of AvroSerializerTraits in AvroRowOutputFormat.h
It is technically possible to put in inside anonymous namespace in the header, while it is usually considered a bad practice. Well, this is mostly about the code, not about forward declarations, but I presume that clang-tidy and company would not happy about this.
Am I missing something?

Reply:
Ok.